crates/autopilot/src/database/onchain_order_events/ethflow_events.rs

```diff
@@ -0,0 +1,176 @@
+use crate::database::events::meta_to_event_index;
+use anyhow::{anyhow, Context, Result};
+use contracts::cowswap_onchain_orders::{
+    event_data::OrderPlacement as ContractOrderPlacement, Event as ContractEvent,
+};
+use database::{
+    ethflow_orders::EthOrderPlacement, events::EventIndex,
+    onchain_broadcasted_orders::OnchainOrderPlacement, orders::Order, PgTransaction,
+};
+use ethcontract::Event as EthContractEvent;
+use std::{collections::HashMap, convert::TryInto};
+
+use super::{OnchainOrderCustomData, OnchainOrderParsing};
+
+pub struct EthFlowOnchainOrderParser {}
```

Comment:
you can also just:
```suggestion
pub struct EthFlowOnchainOrderParser;
```

Its fine as is though.